### PR TITLE
docs: Explain what it is

### LINKED ISSUE #374: Add highlevel documentation of the network-of-terms
Regarding the network-of-terms-api repository as the central repository for the network-of-terms I would expect to find a high level overview of the Network-of-Terms functionality, maybe similar to what is being published in FAQ in the front-end. This would be very helpful for people from outside our network that are interested in our work.

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #374.﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 
+The Network is a **search engine for shared terms**. Given a textual search query, the Network of
+Terms returns terms and their URIs from a growing number of terminology sources. Collection managers
+use these terms to describe their objects. Terms are managed in terminology sources, which must be
+searched and return data in different ways.
+
+The Network of Terms takes care of these differences by offering a **simple search interface**,
+querying the sources in **real-time**, **handling any errors** they return, and **harmonizing the
+results** to the SKOS data model.
+
```
Comment (sdevalk) [inline README.md:15]: Or something like:

> The Network of Terms is a search engine for finding terms in terminology sources, such as thesauri, classification systems, and reference lists. Given a textual search query, the Network of Terms searches one or more terminology sources in real-time and returns the matching terms, including their labels and URIs. The Network of Terms offers a simple search interface, handles errors gracefully if a source does not respond, and harmonizes the results from different sources to the SKOS data model.
> 
> The Network of Terms is intended for everyone who manages heritage information and wants to improve the findability of this information by assigning terms from terminology sources that are used by the institutions in the Dutch Digital Heritage Network.

**Review by EnnoMeijers — CHANGES_REQUESTED** [review]
I agree with the text proposal of Sjors.

File: README.md (hunk 1 above)
Comment (ddeboer) [reply README.md]: Beautiful! Used this version with some minor changes.

**Pull request merged**